beacon_node/lighthouse_network/src/peer_manager/mod.rs

```diff
@@ -732,12 +733,16 @@ impl<E: EthSpec> PeerManager<E> {
                     "peer_id" => %peer_id, "new_seq_no" => meta_data.seq_number());
             }
 
+            let known_custody_group_count = peer_info
+                .meta_data()
+                .and_then(|meta_data| meta_data.custody_group_count().copied().ok());
+
             let custody_group_count_opt = meta_data.custody_group_count().copied().ok();
             peer_info.set_meta_data(meta_data);
 
             if self.network_globals.spec.is_peer_das_scheduled() {
-                // Gracefully ignore metadata/v2 peers. Potentially downscore after PeerDAS to
-                // prioritize PeerDAS peers.
+                // Gracefully ignore metadata/v2 peers.
+                // TODO(das) Potentially downscore after PeerDAS to prioritize PeerDAS peers.
```

Comment:
This should not be possible once peerdas is scheduled, we only send out v3 requests, so maybe this TODO can be removed:

https://github.com/sigp/lighthouse/blob/4894cc2ec7ee8fdc4042c3e53246a4b3534df2a6/beacon_node/lighthouse_network/src/service/mod.rs#L1166-L1174